assets/scripts/admin/statistics.js

```diff
@@ -590,6 +595,28 @@ LS.Statistics2 = function () {
     $(".stats-showpie").click(function () {
         changeGraphType('showpie', this.parentNode);
     });
+
+    var ajaxDownloadStats = function (url, data) {
+        const xhr = new XMLHttpRequest();
+        xhr.open('POST', url, true);
+        xhr.responseType = 'blob';
+        xhr.onload = () => {
+            const contentDisposition = xhr.getResponseHeader('Content-Disposition');
+            const fileName = contentDisposition ? contentDisposition.match(/filename[^;=\n]*=['"](.*?|[^;\n]*)['"]/)[1] : '';
+            if (fileName.length > 0) {
+                // saveAs is implemented by jszip/fileSaver.js
+                saveAs(xhr.response, fileName);
+            } else {
+                ajaxError();
+            }
+            $('#statsContainerLoading').hide();
```

Comment:
Maybe reopen filter again ? Since result show nothing ?
![Peek 20-06-2022 18-09](https://user-images.githubusercontent.com/1439428/174642421-cb6e2b87-5e0c-45d3-9612-5d6bba18a6ed.gif)